articles/protocols/saml/saml-configuration/troubleshoot/auth0-as-sp.md

```diff
@@ -0,0 +1,195 @@
+---
+description: How to troubleshoot SAML-related configuration issues when Auth0 is the service provider
+toc: true
+---
+
+# SAML Troubleshooting When Auth0 is the Service Provider
```

Comment:
`When` -> `when`

I would also reverse the order of the first two words in this and in the previous doc: `Troubleshooting SAML when Auth0 is the Service Provider`